thoth/slo_reporter/sli_apis/sli_user_api.py

```diff
@@ -127,12 +130,14 @@ def _evaluate_sli(self, sli: Dict[str, Any]) -> Dict[str, float]:
 
         return html_inputs
 
-    def _report_sli(self, sli: Dict[str, Any]) -> str:
+    def _report_sli(self, sli: Dict[str, Any], ceph_sli: CephStore) -> str:
         """Create report for User-API SLI.
 
         @param sli: It's a dict of SLI associated with the SLI type.
         """
-        html_inputs = self._evaluate_sli(sli=sli)
+        sli_path = f"{self._SLI_NAME}/{self._SLI_NAME}-{self.configuration.last_week_time}.csv"
+        last_week_data = retrieve_thoth_sli_from_ceph(ceph_sli, sli_path)
+        html_inputs = self._evaluate_sli(sli=sli, last_week_data)
```

Comment:
Step 6. Passing down to `_evaluate_sli`.

Reply:
I would not change `_evaluate_sli` because that method is used to have data stored in a certain way for Hue/Superset and we are not interested in last week results there but only daily as they are right now. 

These changes goes only into the email, therefore to `_report_sli` only, where we need to merge html_inputs obtained with `self.evaluate_sli` and last_week_data :)